apiary.apib

```diff
@@ -2618,6 +2618,124 @@ Your Accredible account is represented as an Issuer within the API.
             }
         }
 
+# Group Data Exports
+
+## Data Exports [/v1/data_exports]
+
+### View All Data Exports [GET]
```

Comment:
We should add a note that this feature must be switched on for use.

Reply:
@alanheppenstall Thank you, please check line 2623